From 379719a2520c1b24536cc07d665aedbabad3ae35 Mon Sep 17 00:00:00 2001
From: gangsaryp <wisnuaryadipa93@gmail.com>
Date: Mon, 24 Aug 2020 13:32:31 +0700
Subject: [PATCH] no message

---
 .env | 5 -----
 1 file changed, 5 deletions(-)
 delete mode 100644 .env

diff --git a/.env b/.env
deleted file mode 100644
index 4d69aeb..0000000
--- a/.env
+++ /dev/null
@@ -1,5 +0,0 @@
-NODE_ENV=developmentd
-PORT=3000
-MONGO_ATLAS_PW=sapakie
-MONGO_ATLAS_USERNAME:wisnuaryadipa
-MONGO_ATLAS_DBNAME:sales
\ No newline at end of file
-- 
2.26.1.windows.1

